scripts/community/boostercreator_injected.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+/** @type {HTMLSelectElement} */
+const gameSelector = document.querySelector( '#booster_game_selector' );
+
+if( gameSelector )
+{
+	// Add a container for the available date before the game selector
+	const availableDateContainer = document.createElement( 'div' );
+	availableDateContainer.id = 'booster_available_date';
+	gameSelector.after( availableDateContainer );
+
+	// Add an event listener to update the available date when the game selector changes
+	gameSelector.addEventListener( 'change', updateBoosterAvailableDate );
+
+	// Initialize the available date when the script loads
+	updateBoosterAvailableDate();
+}
+
+function updateBoosterAvailableDate( )
+{
+	const availableDateContainer = document.getElementById( 'booster_available_date' );
+	const selectedGame = gameSelector.value;
+
+	const availableDate = window.CBoosterCreatorPage.sm_rgBoosterData[ selectedGame ]?.available_at_time;
+
+	if( availableDate )
+	{
+		availableDateContainer.textContent = `You will not be able to create another Booster Pack for this game until ${availableDate}.`;
```

Comment:
This needs localization

Reply:
It is possible on _injected_ files? I've just checked i.e. `tradeoffer_injected.js` and nothing is localized there and when I tried at my own at `boostercreator_injected.js` seemed like `_t( )` function was not available.

Reply:
I think inventory does it, by putting the string onto data structure

Reply:
I don't see what you mentioned, anyway I just need access to `rgBoosterData` for current selected game.
I believe I can move everything into a regular script and send that data on a customEvent on changes.

What do you think about this? see my commit below.

Reply:
https://github.com/SteamDatabase/BrowserExtension/blob/4e7668dc69ec3ee83d9c9e4cf877b17e395b6ed8/scripts/community/profile_inventory.js#L32

 i meant this, but your solution also seems fine.

Reply:
I see, thanks for sharing the knowledge. 

The limitation I see is that my message contains a variable which lives on the `injected` file so I would need to make use of `replace()` anyway at injected file. Saw that you are fine with the `CustomEvent`  path I proposed, so i guess we are good that way.